config/routes.rb

```diff
@@ -7,8 +7,13 @@
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
-  resources :users
+  resources :users do
+    member do
```

Comment:
`member` có ý nghĩa gì em

Reply:
member một phương thức sắp xếp cho các tuyến để phản hồi về các URL chứa id người dùng.

Reply:
chưa đủ, tìm hiểu kỹ hơn, và not vào report của em để khi cần thì dùng nhé
cách nso thay đổi url thế nào...

Reply:
dạ